### PR TITLE
Migrate multitenant dashboard buckets to separate accounts

### PR DESCRIPTION
**Pull request by magul**
Relates saleor/saleor-cloud-infra#1029
Port of #4443 for Saleor 3.17

### CODE REVIEW

File: .github/workflows/prepare-release.yml [hunk 1]
```diff
@@ -68,14 +68,22 @@ jobs:
...
-          aws-secret-access-key: ${{ secrets.AWS_DASHBOARD_STAGING_SECRET_ACCESS_KEY }}
+          aws-access-key-id: ${{ secrets.AWS_CLOUD_STAGING_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_CLOUD_STAGING_SECRET_ACCESS_KEY }}
           aws-region: us-east-1
       - name: Deploy
         run: |
-          aws s3 sync build/dashboard s3://${{ secrets.AWS_STAGING_DEPLOYMENT_BUCKET }}/${ENVIRONMENT}/static/
-          aws s3 cp build/dashboard/index.html s3://${{ secrets.AWS_STAGING_DEPLOYMENT_BUCKET }}/${ENVIRONMENT}/
-          for i in {1..3}; do aws cloudfront create-invalidation --distribution-id ${{ secrets.AWS_STAGING_CF_DIST_ID }} --paths "/dashboard*" && break || sleep 5; done
+          aws s3 sync build/dashboard "s3://${DEPLOYMENT_BUCKET}/${ENVIRONMENT}/static/"
```
Comment (NyanKiyoshi) [inline .github/workflows/prepare-release.yml:76]: `ENVIRONMENT` is not defined anywhere?

Comment (magul) [reply .github/workflows/prepare-release.yml:76]: This is really good question, but beyond my change. @andrzejewsky do we know, what's going on here?

File: .github/workflows/prepare-release.yml [hunk 2]
```diff
@@ -56,6 +56,14 @@ jobs:
     steps:
       - name: Checkout Repo
         uses: actions/checkout@v3
+
+      - name: Set environment
+        # Convert version into staging deployment name (e.g 3.3.0 -> saleor-staging-v33)
+        run: |
+          set -x
+          environment=$(echo $VERSION | sed -n 's#\([0-9]\+\).\([0-9]\+\).*#saleor-staging-v\1\2#p')
```
Comment (NyanKiyoshi) [inline .github/workflows/prepare-release.yml:64]: Seems like a debug leftover?

```suggestion
        run: |
          environment=$(echo $VERSION | sed -n 's#\([0-9]\+\).\([0-9]\+\).*#saleor-staging-v\1\2#p')
```

File: .github/workflows/prepare-release.yml (hunk 1 above)
Comment (NyanKiyoshi) [reply .github/workflows/prepare-release.yml:84]: Resolved at f48f0ce31379d5ceb5710d21f84f80280e5f9121

**Review by NyanKiyoshi — APPROVED** [review]

**Review by andrzejewsky — APPROVED** [review]

**Pull request merged**